diff -aurN xfce4-session-4.10.1.orig/scripts/xflock4 xfce4-session-4.10.1/scripts/xflock4
--- xfce4-session-4.10.1.orig/scripts/xflock4	2013-09-24 22:49:05.473753574 +0200
+++ xfce4-session-4.10.1/scripts/xflock4	2013-09-24 22:51:16.701783286 +0200
@@ -27,7 +27,8 @@
 # Lock by xscreensaver or gnome-screensaver, if a respective daemon is running
 for lock_cmd in \
     "xscreensaver-command -lock" \
-    "gnome-screensaver-command --lock"
+    "gnome-screensaver-command --lock" \
+    "light-locker-command --lock"
 do
     $lock_cmd >/dev/null 2>&1 && exit
 done
